src/index.ts

```diff
@@ -58,16 +50,10 @@ function render(model: Model): HtmlNode<Update> {
   );
 }
 
-/**
- * This function runs functions that require basic rendering of the DOM first.
- *
- * Currently, it is only Chart.js integrations that require this mechanic.
- */
```

Comment:
Do not remove any comments authored by a human. Do not remove any comments written by me.